sippy-ng/src/component_readiness/RegressedTestsPanel.js

```diff
@@ -285,7 +341,16 @@ export default function RegressedTestsPanel(props) {
         componentsProps={{
           toolbar: {
             columns: columns,
-            showQuickFilter: true,
+            addFilters: addFilters,
+            filterModel: filterModel,
+            setFilterModel: setFilterModel,
+            clearSearch: () => requestSearch(''),
+            doSearch: requestSearch,
+            autocompleteData: regressedTests,
+            downloadDataFunc: () => {
+              return filteredTests
+            },
+            downloadFilePrefix: 'regressed_tests',
```

Comment:
Noticed some interesting behavior. If I start in the regressed tests table and filter on a couple components, then go to the Triaged tab, my filters are still in my URL but not in the table for Triaged filters. I can then go back to Untriaged and my filters still apply. It even lets you filter on both and they seem to be preserved for all tabs as you nav around. That works kinda nice in practice.